Rakefile

```diff
@@ -2,6 +2,8 @@ require "bundler/gem_tasks"
 require 'rake/testtask'
 
 Rake::TestTask.new do |t|
+  ENV['RACK_ENV'] = 'test'
```

Comment:
I prefer to move this line like test specific configuration to `minitest_helper.rb`